CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+Version 1.10.6
+==============
+
+This another minor bug fix release in the 1.10 series and is likely
+the last one unless any major issues requiring a hot fix crop up. In
```

Comment:
I would drop 'unless any major issues requiring a hot fix crop up'. We will do what we have to if things go wrong, no need to say that!